README.md

```diff
@@ -1,5 +1,64 @@
 # GearVRf-Demos
 
+## How to run these examples
+* make sure you have [this repository](https://github.com/gearvrf/GearVRf-Demos) and [the GearVR framework](https://github.com/Samsung/GearVRf) in a sibling directory 
+
+```
+git clone https://github.com/Samsung/GearVRf.git
+git clone https://github.com/gearvrf/GearVRf-Demos.git
+```
+
+* if you don't have your computer set up for Android [SDK](https://developer.android.com/studio/index.html) and [NDK](https://developer.android.com/ndk/downloads/index.html) development - the simplest way to get those configured is to use [NVIDIA's Codeworks](https://developer.nvidia.com/codeworks-android) which will automatically configure your `~/.bash_profile`
```

Comment:
not familiar with NVIDIA's Codeworks.  looks like you need to sign up to be an NVidia developer in order to download it.